text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,289 @@
+# Merge Elastic Common Schema with OpenTelemetry Semantic Conventions
+
+## Introduction
+
+This proposal is to merge the Elastic Common Schema (ECS) with the OpenTelemetry Semantic Conventions (SemConv) and provide full interoperability in OpenTelemetry component implementations. We propose to implement this by aligning the OpenTelemetry Semantic Conventions with [ECS FieldSets](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html#ecs-fieldsets) and vice versa where feasible. The long-term goal is to achieve convergence of ECS and OTel Semantic Conventions into a single open schema (e.g. OpenTelemetry Common Schema).
+
+## The Goal
+- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry. To kick off this effort, Elastic will nominate several domain experts to join the OpenTelemetry Semantic Convention Approvers to help with maintaining the new standard.
+- OTel SemConv will adopt ECS in its full scope (except for individual adjustments in detail where inevitable), including the logging, observability and security domain fields, to make the new schema a true successor of ECS and OTel SemConv.
```

Comment:
I notice that metrics is missing from this list and from other references of the ECS addition, but show up in various places like [host fields](https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-cpu-usage). How might ECS metrics, which appear to just be fields, translate to OpenTelemetry metrics which are named and which have attributes as dimensions?

Reply:
I think these are thinks that would need to be worked out together when working on individual domains, attributes and fieldsets. As this proposal rather describes the overall intent, are you OK with leaving these more detailed discussions on certain schema aspects for later?

Reply:
> are you OK with leaving these more detailed discussions on certain schema aspects for later? 

Yes, I approved as is 👍